standard.mk

```diff
@@ -20,7 +20,8 @@ COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-
 CURRENT_COMMIT=$(shell git rev-parse --short=7 HEAD)
 OPERATOR_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT)
 
-OPERATOR_IMAGE_URI=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):v$(OPERATOR_VERSION)
+IMG?=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):v$(OPERATOR_VERSION)
```

Comment:
is the `v` in `:v$` a typo?

Reply:
Thanks a lot to catch that! This wasn't a typo. Actually, I checked working logs for aws-account-operator https://ci-int-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/osd-operators/job/openshift-aws-account-operator-gh-build-master/289/consoleFull and also https://github.com/openshift/aws-account-operator/blob/master/standard.mk#L23. The 'v' is configured as part of this variable but I don't see this used while actual tagging. The IMG variable is as such passed with "managed-upgrade-operator:latest" to `docker build`. The tags for aws-account-operator in https://quay.io/repository/app-sre/aws-account-operator?tab=tags are based on GIT_HASH variable from what I understand. For now, have removed 'v' since the working logs didn't refer it. We can add it in future if required.